patches/dotenv-expand@9.0.0.patch

```diff
@@ -1,8 +1,19 @@
 diff --git a/lib/main.js b/lib/main.js
-index c873cc77229d4cd0cf9de98ae0970b25d89f312f..ddf570558e985760efde52af37a41b56282d30a6 100644
+index c873cc77229d4cd0cf9de98ae0970b25d89f312f..7681ae0f2d843f0b9466f0f84799af56cd178500 100644
 --- a/lib/main.js
 +++ b/lib/main.js
-@@ -50,9 +50,10 @@ function expand (config) {
+@@ -17,6 +17,10 @@ function _interpolate (envValue, environment, config) {
+       replacePart = parts[0]
+       value = replacePart.replace('\\$', '$')
+     } else {
++      // compatible with env variables ended with unescaped $
```

Comment:
```suggestion
+      // PATCH: compatible with env variables ended with unescaped $
```

would be nice to add the `PATCH:` comment to be consistent with the other patch comment.